lib/rules/function-url-no-scheme-relative/README.md

```diff
@@ -9,6 +9,13 @@ a { background-image: url('//www.google.com/file.jpg'); }
  *  This scheme-relative url */
 ```
 
+<!-- prettier-ignore -->
+```css
+@import url('//www.google.com/file.jpg');
+/**         ↑
+ *  This scheme-relative url */
+```
+
```

Comment:
not necessary
i.e. one example is the norm

Reply:
Agree, please remove this example.